backend/src/test/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/mapper/VerkuendungResponseMapperTest.java

```diff
@@ -0,0 +1,195 @@
+                                          <meta:fna>754-28-1</meta:fna>
+                                          <!-- Die vorliegende Angabe von meta:gesta besitzt keine fachliche Korrektheit. -->
+                                          <meta:gesta>nicht-vorhanden</meta:gesta>
+                                          <!-- Die vorliegenden Angaben von meta:federfuehrung besitzen keine fachliche Korrektheit. -->
+                                          <meta:federfuehrung>
+                                            <meta:federfuehrend ab="2002-12-01" bis="unbestimmt">Bundesministerium des Innern und für Heimat</meta:federfuehrend>
+                                            <meta:federfuehrend ab="2002-10-01" bis="2002-12-01">Bundesministerium der Justiz</meta:federfuehrend>
+                                          </meta:federfuehrung>
+                                        </meta:legalDocML.de_metadaten>
+                                      </akn:proprietary>
+                    </akn:meta>
+
+                    <akn:preface eId="einleitung-1" GUID="fc10e89f-fde4-44bf-aa98-b6bdea01f0ea">
+                     <akn:longTitle eId="einleitung-1_doktitel-1" GUID="abbb08de-e7e2-40ab-aba0-079ce786e6d6">
+                        <akn:p eId="einleitung-1_doktitel-1_text-1" GUID="3e7c2134-d82c-44ba-b50d-bad9790375a0">
+                           <akn:docStage eId="einleitung-1_doktitel-1_text-1_docstadium-1" GUID="3b355cab-ce10-45b5-9cde-cc618fbf491f" />
+                           <akn:docProponent eId="einleitung-1_doktitel-1_text-1_docproponent-1" GUID="c83abe1e-5fde-4e4e-a9b5-7293505ffeff" />
+                           <akn:docTitle
+                              eId="einleitung-1_doktitel-1_text-1_doctitel-1" GUID="8c4eabab-9893-455e-b83b-c46f2453f2fb">Gesetz zur Regelung des öffentlichen Vereinsrechts</akn:docTitle>
+                           <akn:shortTitle eId="einleitung-1_doktitel-1_text-1_kurztitel-1" GUID="fdb8ed28-2e1f-4d81-b780-846fd9ecb716">( <akn:inline
+                                 eId="einleitung-1_doktitel-1_text-1_kurztitel-1_inline-1" GUID="bdff7240-266e-4ff3-b311-60342bd1afa2" refersTo="amtliche-abkuerzung" name="attributsemantik-noch-undefiniert">Vereinsgesetz</akn:inline>)</akn:shortTitle>
+                        </akn:p>
+                     </akn:longTitle>
+                     <akn:block eId="einleitung-1_block-1" GUID="010d9df0-817a-49b6-a121-d0a1d412a3e3" name="attributsemantik-noch-undefiniert">
+                        <akn:date eId="einleitung-1_block-1_datum-1" GUID="28fafbe4-403d-4436-8d0d-7241cbbdade0" refersTo="ausfertigung-datum" date="1964-08-05">Vom 5. August 1964 </akn:date>
+                     </akn:block>
+                  </akn:preface>
+                 </akn:act>
+              </akn:akomaNtoso>
```

Comment:
Could you move this to a fixture file? That makes updating it for changes in future ldml_de versions a lot simpler

Reply:
@malte-laukoetter could you explain why would it be simpler? It would be anyway a find+replace for both scenarios right? Whether it is a fixture file (XML format) or just simple template string within a test

Reply:
for the update from 1.6 to 1.7 we used a xslt file to do the transformation of the more complex changes that could not be achieved using a simple search-and-replace. And for doing that we need a xml file as an input.
For the eId changes for the 1.8 update I also imagine that we will need a similar solution.